Document the `logout` commands to revoke authentication tokens. (#785)
digital-asset/docs.daml.com: docs/3.1/docs/canton/usermanual/apis.rst

However, we suggest keeping the default values.

As mentioned above, a generated token allows the member that provides it during a call to authenticate on public API

> services. Therefore, these tokens are sensitive information that must not be disclosed. If an operator suspects that
> the authentication token for a member has been leaked or somehow compromised, thereby allowing an illegitimate client
> to impersonate that member, they should use the ``logout`` console command to immediately revoke all valid tokens of
> that member and close the sequencer connections. The legitimate member will then automatically reconnect and obtain

Reviewer: ```suggestion
services. Therefore, these tokens are sensitive information that must not be disclosed. If an operator suspects that
the authentication token for a member has been leaked or somehow compromised, they should use the ``logout`` console command to immediately revoke all valid tokens of
that member and close the sequencer connections. The legitimate member automatically reconnects and obtains
```

new tokens through the challenge-response protocol described above.

> The command to use is slightly different whether the member is a participant or a mediator, e.g.:

Reviewer: ```suggestion
The command is slightly different depending on whether the member is a participant or a mediator, for example:
```
